.travis.yml

```diff
@@ -45,19 +45,21 @@ script:
   - 'sudo docker exec --tty "$(cat ${container_id})" env TERM=xterm ansible all -i "localhost," -c local -m setup'
 
   # Install dependencies
-  - 'sudo docker exec --tty "$(cat ${container_id})" env TERM=xterm ansible-galaxy install geerlingguy.java'
+  - 'sudo docker exec --tty "$(cat ${container_id})" env TERM=xterm ansible-galaxy install geerlingguy.java,1.7.4'
```

Comment:
@Rudd-O so 1.7.4 is the latest version (the default). I'ld try with 1.7.2, which is latest one without that 'apt cache update' task removal